samples/node/credentials.js

```diff
@@ -0,0 +1,6 @@
+const credentials = {
```

Comment:
Why not just `export = {}` so that you can do `const credentials = require('./credentials');`?